lib/nimbusRecipeCollection.ts

```diff
@@ -33,4 +36,38 @@ export class NimbusRecipeCollection implements NimbusRecipeCollectionType {
 
     return this.recipes
   }
+
+  /**
+   * @returns a list of RecipeInfo with updated ctrPercent properties for all
+   * the branches of every recipe in this collection
+   */
+  async getExperimentAndBranchInfos(): Promise<RecipeOrBranchInfo[]> {
```

Comment:
I decided to move this function here because Next.js was complaining about multiple exports coming from `page.tsx` and I had the function exported for testing purposes.

Reply:
Very reasonable.